armi/cases/tests/test_cases.py

```diff
@@ -212,6 +212,41 @@ def test_run(self):
                 self.assertIn("xsGroups", mock._outputStream)
                 self.assertIn("Completed EveryNode - cycle 0", mock._outputStream)
 
+    def test_clone(self):
+        testTitle = "CLONE_TEST"
+        # test the short write style
+        with directoryChangers.TemporaryDirectoryChanger():
+            cs = settings.Settings(ARMI_RUN_PATH)
+            setMasterCs(cs)
+            case = cases.Case(cs)
+            shortCase = case.clone(
+                additionalFiles=["ISOAA"],
+                title=testTitle,
+                modifiedSettings={"verbosity": "important"},
+            )
+            # Check additional files made it
+            self.assertTrue(os.path.exists("ISOAA"))
+            # Check title change made it
+            clonedYaml = testTitle + ".yaml"
+            self.assertTrue(os.path.exists(clonedYaml))
+            self.assertTrue(shortCase.title, testTitle)
+            # Check on some expected settings, modified and removed with the write style
+            txt = open(clonedYaml, "r").read()
+            self.assertNotIn("availabilityFactor", txt)
+            self.assertIn("verbosity: important", txt)
+
+        # test the medium write style
+        with directoryChangers.TemporaryDirectoryChanger():
+            cs = settings.Settings(ARMI_RUN_PATH)
+            setMasterCs(cs)
+            case = cases.Case(cs)
+            case.clone(writeStyle="medium")
+            clonedYaml = "armiRun.yaml"
+            self.assertTrue(os.path.exists(clonedYaml))
+            # validate a default value was removed
+            txt = open(clonedYaml, "r").read()
+            self.assertIn("availabilityFactor", txt)
```

Comment:
I would add a comment in here as to why we would expect "availabilityFactor" to be in the written CS (i.e. because it is in the original CS but at it's default value, so it won't be in the short dump, but it will be in the medium dump)

Without seeing the original CS, it is not so obvious as to what is going on here.